js/react_native/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "onnxruntime-react-native",
+  "version": "1.7.0",
+  "description": "Onnxruntime bridge for react native",
+  "main": "dist/commonjs/index",
+  "module": "dist/module/index",
+  "types": "dist/typescript/index.d.ts",
+  "react-native": "lib/index",
+  "source": "lib/index",
+  "files": [
+    "lib",
+    "dist",
+    "android",
+    "ios",
+    "onnxruntime-react-native.iphoneos.podspec",
+    "onnxruntime-react-native.iphonesimulator.podspec",
+    "!dist/typescript/example",
+    "!android/build",
+    "!ios/build",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__",
+    "!.gitignore",
+    "!.placeholder"
+  ],
+  "scripts": {
+    "test": "jest",
+    "typescript": "tsc --noEmit",
+    "prepare": "bob build",
+    "release": "release-it",
+    "example": "yarn --cwd example",
+    "pods": "cd example && pod-install --quiet",
+    "bootstrap": "yarn example && yarn && yarn pods",
+    "prepack": "node ./scripts/prepack"
+  },
+  "keywords": [
+    "react-native",
+    "ios",
+    "android",
+    "ONNX",
+    "ONNXRuntime",
+    "ONNX Runtime"
+  ],
+  "repository": "https://github.com/Microsoft/onnxruntime.git",
+  "author": "hanbitmyths",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/onnxruntime/issues"
+  },
+  "homepage": "https://github.com/Microsoft/onnxruntime/js/react_native#readme",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "devDependencies": {
+    "@release-it/conventional-changelog": "^2.0.0",
+    "@types/jest": "^26.0.0",
+    "@types/react": "^16.9.19",
+    "@types/react-native": "0.62.13",
+    "jest": "^26.0.1",
+    "pod-install": "^0.1.0",
+    "prettier": "^2.0.5",
+    "react": "16.13.1",
+    "react-native": "0.63.4",
+    "react-native-builder-bob": "^0.17.1",
+    "release-it": "^14.2.2",
+    "typescript": "^4.1.3"
+  },
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  },
+  "jest": {
+    "preset": "react-native",
+    "modulePathIgnorePatterns": [
+      "<rootDir>/example/node_modules",
+      "<rootDir>/dist/"
+    ]
+  },
+  "release-it": {
+    "git": {
+      "commitMessage": "chore: release ${version}",
+      "tagName": "v${version}"
+    },
+    "npm": {
+      "publish": true
+    },
+    "github": {
+      "release": true
+    },
+    "plugins": {
+      "@release-it/conventional-changelog": {
+        "preset": "angular"
+      }
+    }
+  },
+  "react-native-builder-bob": {
+    "source": "lib",
+    "output": "dist",
+    "targets": [
+      "commonjs",
+      "module",
+      [
+        "typescript",
+        {
+          "project": "tsconfig.build.json"
+        }
+      ]
+    ]
+  },
+  "dependencies": {
+    "buffer": "^6.0.3",
```

Comment:
It looks like only backend.ts uses buffer, and all usage should be able to be replaced by `ArrayBuffer` and `Uint8Array`. We can drop this dependency.

Also, other files under `example` folder uses buffer as well. the package.json under that folder should list buffer as dependency, not in this file. (according to how the code uses buffer, it's highly possible to drop the dependency there as well)

Reply:
a user using this package needs this dependency like onnxruntime-common.